apps/backend/src/transaction/payment.service.ts

```diff
@@ -31,12 +31,14 @@ export class PaymentService {
           location_id: location.location_id
         },
         status: paymentStatus,
-        method: In(pos_payment_methods)
+        payment_method: {
```

Comment:
This is a wrong approach, linking two unrelated domain models. 

TDI 34 has nothing to do with payments looking objectively from a transactions point of view. This creates unnecessary coupling.

Reply:
Resolved as per conversation